src/app/shell/personal-cabinet/provider/create-workshop/create-additional-about-form/create-additional-about-form.component.html

```
@@ -146,7 +146,11 @@
   <!--    </mat-select>-->
   <!--  </mat-form-field>-->
   <label class="step-label">{{ 'FORMS.LABELS.GROUP_TYPE' | translate }}<span class="step-required">*</span></label>
-  <mat-form-field appearance="fill">
+  <mat-form-field
+    appearance="fill"
+    [matTooltip]="'FORMS.TOOLTIP.GROUP_TYPE_IS_DISABLED' | translate"
+    [matTooltipDisabled]="!isMinSportSelected"
+    >>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)